source/linear-algebra/source/05-GT/05.ptx

```diff
@@ -0,0 +1,233 @@
+        </p>
+    </introduction>
+    <task>
+        <p>
+            Calculate <m>M_{\mathcal{B}}</m> using technology. 
+        </p>
+        <sage language="octave">
+            <input>
+                
+            </input>
+            <output>
+                
+            </output>
+        </sage>
+    </task>
+    <task>
+        <p>
+            Use your result to calculate <m>C_\mathcal{B}\left(\begin{bmatrix}1\\1\\1\end{bmatrix}\right)</m> and express the vector <m>\begin{bmatrix}1\\1\\1\end{bmatrix}</m> as a linear combination of <m>\vec{v}_1,\vec{v}_2,\vec{v}_3</m>.
+        </p>
+    </task>
+</activity>
+<observation>
+    <p>
+        Let <m>T\colon\IR^n\to\IR^n</m> be a linear transformation and let <m>A</m> denote its standard matrix. 
+        If <m>\cal{B}=\setList{\vec{v}_1,\dots, \vec{v}_n}</m> is some other basis, then we have:
+        <md>
+            <mrow>M_\mathcal{B}AM_{\mathcal{B}}^{-1} \amp= M_\mathcal{B}A[\vec{v_1}\cdots\vec{v}_n] </mrow>
+            <mrow> \amp= M_\mathcal{B}[T(\vec{v}_1)\cdots T(\vec{v}_n)]</mrow>
+            <mrow> \amp= [C_\mathcal{B}(T(\vec{v}_1))\cdots C_\mathcal{B}(T(\vec{v}_n))]</mrow>
+        </md>
+        In other words, the matrix <m>M_{\mathcal{B}}AM_{\mathcal{B}}^{-1}</m> is the matrix whose columns consist of <em><m>\mathcal{B}</m>-coordinate</em> vectors of the image vectors <m>T(\vec{v}_i)</m>.
+        The matrix <m>M_{\mathcal{B}}AM_{\mathcal{B}}^{-1}</m> is called the <alert>matrix of <m>T</m> with respect to <m>\mathcal{B}</m>-coordinates</alert>.
+    </p>
+</observation>
+
+<activity>
+    <introduction>
+        <p>
+            Let <m>\mathcal{B}=\setList{\vec{v}_1,\vec{v}_2,\vec{v}_3}=\setList{\begin{bmatrix}1\\-2\\1\end{bmatrix},\begin{bmatrix}-1\\0\\3\end{bmatrix},\begin{bmatrix}0\\1\\-1\end{bmatrix}}</m> be basis from the previous Activity. 
+            Let <m>T</m> denote the linear transformation whose standard matrix is given by:
+            <me>A=\begin{bmatrix}9&amp;4&amp;4\\6&amp;9&amp;2\\-18&amp;-16&amp;-9\end{bmatrix}.</me>
+        </p>
+    </introduction>
+    <task>
+        <p>
+            Calculate the matrix <m>M_\mathcal{B}AM_{\mathcal{B}}^{-1}</m>.
+        </p>
+        <sage language="octave">
+            <input>
+                
+            </input>
+            <output>
+                
+            </output>
+        </sage>
+    </task>
+    <task>
+        <p>
+            The matrix <m>A</m> describes how the standard basis of <m>\IR^3</m> is transformed by the linear transformation <m>T</m>.
+            The matrix <m>M_\mathcal{B}AM_{\mathcal{B}}^{-1}</m> describes how <m>\cal{B}</m> is transformed (in <m>\mathcal{B}</m>-coordinates). 
+            Which of these two descriptions is easier for you to describe/understand/visualize and why?
```

Comment:
I'm not sure if this is asking the students to compare the two descriptions of <m>A</m> and `<m>M_\mathcal{B}AM_{\mathcal{B}}^{-1}</m>`, or if it's asking them to compare this description of `<m>M_\mathcal{B}AM_{\mathcal{B}}^{-1}</m>` with the one from the previous Observation.

Reply:
I don't want/need them to link this back to the observation. I'm basically ask them to try describing how T transforms the standard basis and then to describe how T transforms the B-basis and compare/contrast.

Suggestion for a re-phrasing?